### DISCUSSION TITLE
only use filetype comment strings?

### DISCUSSION THREAD

**Discussion by wadegbow** [numToStr/Comment.nvim#455] [Q&A]
Is there a way to only use the configured comment strings for a certain filetype and not use whatever Treesitter thinks the current context is?

For example, in a Twig file I would like to only use twig style `{# %s #}` comments and never use HTML style `<!-- %s -->` comments. Most of the time the context based comments work great but in my Twig files I would like to be able to set it to only use Twig style comments. Is this possible?

Hopefully that makes sense, might be confusing some of the terminology. Thanks for any help!

**Comment by sharpchen** [comment]
I think you can add an autocmd on `FileType` for `twig` to override html commentstring; And add another autocmd to set it back on `BufLeave`
